Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * AsmParser: Parsing rules for source location comments have been relaxed: Whitespace between the indices as well as single-quoted code snippets are now allowed.
```

Comment:
I guess we should also mention that it fixes that regexp segfault: https://github.com/ethereum/solidity/issues/13496.

Reply:
I see you appended it at the end. I think it should have its own entry. It's a proper bugfix.

Also, the original entry about relaxed debug comment parsing should really go under `Language Features:`, with a `Yul: ` prefix. I think we can consider these comments a part of Yul syntax. At least we're parsing them along with the rest of Yul code.